### ISSUE TITLE
Array validations in declaration request

### ISSUE THREAD

**Issue by DPashchenko** [Nebo15/ehealth.api#564]
Service: [POST] /api/declaration_requests

Add logic:
```
if $.declaration_request.person.confidant_person.length > 0
  check objects in array; at least one object should contain:
  $.declaration_request.person.confidant_person[:].relation_type === "PRIMARY" 
else
  else return error 422 "one and only one confidant_person with type PRIMARY is required"
```

2. Check $.declaration_request.person.addresses
It should contain:

    * One object with $.declaration_request.person.addresses[:].type === "REGISTRATION"
Else return error 422 "one and only one address with type REGISTRATION is required"
    * One object with $.declaration_request.person.addresses[:].type === "RESIDENCE"
Else return error 422 "one and only one address with type RESIDENCE is required"
    * If no objects with types above, return 422 "one address with type RESIDENCE and one address with type REGISTRATION is required"

**Comment by AlekzandrV** [comment]
@gmile Подобную проверку делали ранее. https://github.com/Nebo15/ehealth.api/blob/master/lib/ehealth/validators/addresses.ex

**Comment by gmile** [comment]
This is done in de73ccca3b9c5ff3f2a48e7c3568f7c2e8c12df6 and 8a43abba04318e088a83eff0c39c79ab7d702f59 respectively. Available starting version 0.3.63, which was just deployed.
